lib/lotus/commands/new/abstract.rb

```diff
@@ -0,0 +1,204 @@
+require 'lotus/utils/hash'
+module Lotus
+  module Commands
+    class New < Thor
+      class Abstract
```

Comment:
Instead dumping all this database logic here wouldn't be better to put that in a specific class that does only this? I did it in #325 

Also, is there any change to break this huge PR into smaller ones focused in one thing. I have the feeling that if we keep this approach of changing everything in a only huge step it'll take forever, also it's very difficult to review and prone to conflict/block other improvements.